src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/RSAOAEPKeyExchangeFormatter.cs

```diff
@@ -47,6 +47,11 @@ public override string Parameters
             get {return null;}
         }
 
+        public RandomNumberGenerator Rng {
+            get { return null; }
+            set { throw new PlatformNotSupportedException(); }
```

Comment:
My original thoughts were "the RNG properties are stupid, nothing ever reads them", so it then becomes "why pay the cost of the field?", assuming these behaviors came from notes I made.

But I guess it's easy enough to just blind port the desktop code and be done with it.